Library/Formula/dirmngr.rb

```diff
@@ -18,6 +18,11 @@ class Dirmngr < Formula
   depends_on "libksba"
   depends_on "pth"
 
+  patch :p0 do
+    url "http://bugs.g10code.com/gnupg/file419/dirmngr-pth-fix.patch"
+    sha256 "0efbcf1e44177b3546fe318761c3386a11310a01c58a170ef60df366e5160beb"
+  end
```

Comment:
The patch is written by the upstream developer http://bugs.g10code.com/gnupg/issue1590
but it seems dirmngr has been unmaintained for years no one seems to be in charge for releasing a new version.
It is trivial to show that dirmngr is completely broken in homebrew: just install dirmngr and run dirmngr -> segmentation fault.
I'm sorry, but I cannot spare more time for such a simple fix for such an obvious and well-known bug.